codegen/templates/code/types/json.gen.gotmpl

```diff
@@ -19,7 +19,10 @@ var _ = math.Inf
 
 var (
 	marshaller = &skv2jsonpb.Marshaler{}
-	unmarshaller = &jsonpb.Unmarshaler{}
+	unmarshaller = &jsonpb.Unmarshaler{
+		AllowUnknownFields: true,
```

Comment:
if we made a breaking api change (we shouldn't do that, but let's say that we did), and say changed the type of a field from a string to a boolean, and then we tried to unmarshal a resource that was still using a string value, would unmarshaling fail, or would that field be treated as an 'unknown' field and unmarshaling would succeed? (i would like to find a way for unmarshaling to succeed even on breaking api changes like this)

Reply:
TBH I'm not sure, that's not the purpose of this change. We need to remove an unecessary status field in GM, and this will allow that.

Reply:
unmarshaling would fail in that case.

allow unknown field specificallly handles the case when an unknown field is discovered.

your example is a known field which will fail to unmarhsal with error akin to `cannot unmarshal field of type X into type Y`